test/integration/suite_test.go

```diff
@@ -36,9 +37,31 @@ func TestMain(m *testing.M) {
 
 	fmt.Println("INFO: setting up test environment")
 	kongbuilder := kong.NewBuilder()
+	kongAdminPwd := ""
+	extraControllerArgs := []string{}
+	if kongEnterpriseEnabled == "true" {
+		licenseJSON := os.Getenv("KONG_LICENSE_DATA")
```

Comment:
non-blocker: it seems like we've been using `KONG_ENTERPRISE_LICENSE` in secrets and so forth, but we're consuming it here as `KONG_LICENSE_DATA`. In retrospect if you've got some spare time seems like we should make this uniform and just make it `KONG_LICENSE_DATA` in the github environment as well.

Reply:
See the last special note: I would also like to change this, but I don't have access to repo admin to change the secret name there, so I just went with something in my power that did work.

If you want to change it and do have admin, go ahead and do so along with a commit to change the name in code.